markdown/report.Rmd

```diff
@@ -0,0 +1,416 @@
+
+The proportion of time spent at home or in a community setting towards the end of life provides a high level indication of progress in implementation of the national action plan. This is inferred by measuring the amount of time spent in a hospital setting during the last months of life (using hospital admissions data) and from this estimating the time spent at home or in a community setting.   
+
+It is envisaged that an increase in this measure will reflect both quality and value through more effective, person centred and efficient end of life care with people being better able to be cared for at home or closer to home with a planned approach to end of life care resulting in less time in a hospital setting. 
+
+Over time, an increase in the proportion of time spent at home / in a community setting at the end of life may indicate an improvement in the quality and efficiency of palliative care. This increase would be expected in the medium to long term as a result of an increase in the use of anticipatory care plans and electronic palliative care summaries. 
+
+This measure is based on Acute Inpatient and Day Case and Geriatric Long Stay (SMR01/SMR01 GLS) and Mental Health Inpatient and Day Case (SMR04), returns covering non-obstetric hospitals in Scotland. 
+<Div custom-style = "Heading 3_pgbrk">Appendix * - Data Quality and Completeness</div>
+
+Hospital data and National Records of Scotland deaths data are provisional for 2018/19. Figures for the latest year should therefore be treated with caution as there could be an undercount of hospital admissions, particularly for people who died in the last three months of the year. 
+
+Estimated completeness figures for Acute Inpatient and Day Case records (SMR01) are presented in the following table. Note that there are particular issues with completeness for NHS Forth Valley and NHS Greater Glasgow & Clyde data for the last quarter (Quarter 4) of 2018/19 and so data for these areas in particular should be treated with caution.
+
+#### SMR01 Completeness Table HERE
+
+This publication will be updated in October 2019 with final figures.
+
+<Div custom-style = "Heading 3_pgbrk">Appendix 3 - Methodology</div>
+
+This publication includes data from financial years 2010/11 to 2018/19 at Scotland, Health Board and Health and Social Care Partnership level. 
+
+In the August 2016 release the methodology for how to measure the: ‘percentage of the last six months of life spent at home or in a community setting’ was revised compared to previous publications. Therefore results presented in this publication will not be comparable to figures published prior to August 2016. However, trend data from 2010/11 to 2018/19 is available using the revised methodology in this publication.
+
+**Calculation of Measure**
```

Comment:
Type a space after the last asterisk otherwise the text after it won't appear as a new line.